packages/data/src/registry.js

```diff
@@ -336,17 +262,21 @@ export function createRegistry( storeConfigs = {} ) {
 	 * current internal representation of the registry, which may be enhanced
 	 * by plugins.
 	 *
-	 * @param {Object<string,Function>} fns Object of function values.
+	 * @param {Object<string,Function>} attributes Object of function values.
 	 *
 	 * @return {Object<string,Function>} Object enhanced with plugin proxying.
 	 */
-	function withPlugins( fns ) {
-		return mapValues( fns, ( fn, key ) => function() {
+	function withPlugins( attributes ) {
+		return mapValues( attributes, ( attribute, key ) => function() {
+			if ( key === 'attributes' ) {
```

Comment:
What is `attributes` ? Is this meant to be `namespaces` ? Or better, could we just have this be an `typeof key !== 'function'` condition?